plugins/org.eclipse.fordiac.ide.fbtypeeditor/src/org/eclipse/fordiac/ide/fbtypeeditor/editors/FBTypeEditor.java

```diff
@@ -157,7 +152,7 @@ private void createSaveDialog(final IProgressMonitor monitor) {
 		// Depending on the button clicked:
 		switch (fbSaveDialog.open()) {
 		case DEFAULT_BUTTON_INDEX:
-			doSaveWithUpdate(monitor, fbSaveDialog.getDataHandler().getCollectedElements());
+			doSaveInternal(monitor);
```

Comment:
I think this should be executed no matter what the user selects

Reply:
No because if the user selects cancel we must not save.

Reply:
Ok, I thought the requirements were that it would be saved no matter what the user said. But maybe I'm remembering that wrong.